src/Symfony/Component/ErrorHandler/BufferingLogger.php

```diff
@@ -22,12 +22,12 @@ class BufferingLogger extends AbstractLogger
 {
     private $logs = [];
 
-    public function log($level, $message, array $context = [])
+    public function log($level, $message, array $context = []): void
```

Comment:
`int $level`? `string $message`?

Reply:
`vendor/psr/log/Psr/Log/LoggerInterface.php` (which is extended in this case through the `AbstractLogger`) has the same method signature, needs to be used here too.